src/screens/AccountScreens/ProfileScreen/index.js

```diff
@@ -1,11 +1,56 @@
 import React from 'react';
 import { styles } from './style';
-import { View, Text } from 'react-native';
+import { View, Text, Image, } from 'react-native';
+import { Button, Input, Loading } from '../../../components';
+import { useAuthStore } from '../../../shared/zustand/auth';
+import { THEME_COLORS } from '../../../components/theme';
+
+
 
 const Profile = () => {
+  const { isLoading } = useAuthStore();
+  if (isLoading) {
+    return <Loading />;
+  }
   return (
     <View style={styles.container}>
-      <Text style={{fontSize: 30,}}>Profile Page</Text>
+      <Image
+        source={{
+          uri: "https://w.wallhaven.cc/full/k7/wallhaven-k7xwjm.jpg"
+        }}
+        style={styles.image}
+      ></Image>
+      <View style={styles.userInfoContainer}>
+        <View  style={styles.userInfoTextContainer}>
+        <Text style={styles.userInfoText}>
+          User's Email
```

Comment:
The value of this text element should come from a state using React's state hook. You can name the state as something like `userEmailString`. This makes it easier for data integration.